Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -2804,7 +2825,7 @@ internal static async Task ListenBucketNotificationsAsync_Test1(IMinioClient min
                     }
                 }
 
-            // subscription.Dispose();
+            subscription.Dispose();
```

Comment:
Why not use the `using` keyword when getting the subscription? That would automatically dispose the subscription when it goes out of scope.

Reply:
Done!